src/modules/likesCounter.js

```diff
@@ -0,0 +1,36 @@
+const getAllLikes = async () => {
+  const appkey = localStorage.getItem('apikey');
+  const url = `https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/${appkey}/likes/`;
+  let likes = await fetch(url);
+  likes = await likes.json();
+  return likes;
+};
+
+const addLike = async (id) => {
+  const appkey = localStorage.getItem('apikey');
+  const url = `https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/${appkey}/likes/`;
+  const rawResponse = await fetch(url, {
+    method: 'POST',
+    headers: {
+      Accept: 'application/json',
+      'Content-Type': 'application/json',
+    },
+    body: JSON.stringify({ item_id: id }),
+  });
+  return rawResponse;
+};
+
+const addLikeEvent = () => {
+  const likebtn = document.querySelectorAll('.likes_wrap');
+  likebtn.forEach((btn) => {
+    btn.addEventListener('click', () => {
+      const id = btn.getAttribute('id');
+      addLike(id);
+      const num = btn.querySelector('.counter');
+      const newNum = parseInt(num.innerText, 10) + 1;
+      btn.querySelector('.counter').innerText = newNum;
+    });
+  });
+};
+
+module.exports = { addLikeEvent, getAllLikes };
```

Comment:
- Great use of camel cases as they are one of the best Javascript practices 👍🏼 
- Great use of DRY : 

> The code was kept as simple as it was possible in order to make sure the codebase is easier to navigate.
No repeating in code sequences for the easy flow and easy readability of the code.
Implicit Return
Default Parameters